services/repository/src/main/java/org/sagebionetworks/repo/web/filter/SessionIdCookieSetterFilter.java

```diff
@@ -29,7 +29,6 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
 			//add cookie to the response
 			Cookie sessionIdCookie = new Cookie(HttpRequestIdentifierUtils.SESSION_ID_COOKIE_NAME, UUID.randomUUID().toString());
 			sessionIdCookie.setHttpOnly(true);
-			sessionIdCookie.setSecure(true);
```

Comment:
This is supposed to make the client only remember the cookie if they are contacting via HTTPS. It is not necessary for our use case of the sessionID and makes local testing impossible since that's done over HTTP.